kubernetes/cosmos/templates/rclone-cronjob.yaml

```diff
@@ -71,7 +83,7 @@ spec:
             emptyDir: {}
 {{- end }}
 {{- end -}}
-{{- if .Values.rclone.initialIngestion }}
+{{- if and .Values.rclone.initialIngestion .Release.IsInstall }}
```

Comment:
`IsInstall` doesn't seem to work E2E with the operator. Will have to revert this part back.

Reply:
Added the `IsInstall` but turned the job into a post-install hook. This way the job isn't immutable and only gets ran at install time and not at every upgrade.